src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -39,4 +39,24 @@ class LeoRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest with
       status shouldEqual StatusCodes.NotFound
     }
   }
+
+  it should "202 when deleting a cluster" in {
```

Comment:
Should probably be `in isolatedDbTest {`

I know you're creating a cluster and deleting it, but if something fails the DB could still be left in a bad state. Safer to use isloatedDbTest